odl/contrib/pytorch/modules/basic.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Basic pytorch modules for general-purpose use."""
+
+import torch
+from torch import nn
+
+__all__ = ('Logistic', 'InvLogistic')
+
+
+class Logistic(nn.Module):
```

Comment:
Not at all sure if ODL should have stuff like this that we expose "externally" so to say. If we need it internally, it should be in some util folder.

We don't want to end up as a general purpose pytorch library (i guess?)

For TF I'm hosting my stuff at
https://github.com/adler-j/adler